### PR TITLE
Updated

### PR DESCRIPTION
**Pull request by DevendarReddyDarukumalli**

### CODE REVIEW

**Review by gwolf2u — CHANGES_REQUESTED** [review]
Please align formatting with rest of the XML

File: update.xml [hunk 1]
```diff
@@ -746,4 +746,17 @@
             <telegram>https://t.me/rasenkai</telegram>
         </z2_plus>
     </manufacturer>
+	<manufacturer id="Smartron">
```
Comment (gwolf2u) [inline update.xml]: remove indentation a bit here

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: update.xml (hunk 1 above)
Comment (DevendarReddyDarukumalli) [reply update.xml]: removed

Comment (gwolf2u) [reply update.xml]: no it's not
remove indentation means to lower the spaces 
if you see starting of manufacturer tag is not aligned with ending

Comment (DevendarReddyDarukumalli) [reply update.xml]: now its removed

**Review by gwolf2u — APPROVED** [review]

**Pull request closed without merging**